rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -390,18 +390,30 @@ namespace
 		return result;
 	}
 
-	std::array<VkRenderPass, 120> get_precomputed_render_passes(VkDevice dev, const vk::gpu_formats_support &gpu_format_support)
+	std::array<VkRenderPass, 120> get_precomputed_render_passes(VkPhysicalDevice gpu, VkDevice dev, const vk::gpu_formats_support &gpu_format_support)
 	{
 		std::array<VkRenderPass, 120> result = {};
+		VkImageFormatProperties props = {};
 
 		const std::array<VkFormat, 3> depth_format_list = { VK_FORMAT_UNDEFINED, VK_FORMAT_D16_UNORM, gpu_format_support.d24_unorm_s8 ? VK_FORMAT_D24_UNORM_S8_UINT : VK_FORMAT_D32_SFLOAT_S8_UINT };
 		const std::array<VkFormat, 8> color_format_list = { VK_FORMAT_R5G6B5_UNORM_PACK16, VK_FORMAT_B8G8R8A8_UNORM, VK_FORMAT_R16G16B16A16_SFLOAT, VK_FORMAT_R32G32B32A32_SFLOAT, VK_FORMAT_R8_UNORM, VK_FORMAT_R8G8_UNORM, VK_FORMAT_A1R5G5B5_UNORM_PACK16, VK_FORMAT_R32_SFLOAT };
 
-
 		for (const VkFormat &color_format : color_format_list)
 		{
+			VkResult support = vkGetPhysicalDeviceImageFormatProperties(gpu, color_format, VK_IMAGE_TYPE_2D, VK_IMAGE_TILING_OPTIMAL, VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT, 0, &props);
+			if (support != VK_SUCCESS)
+			{
+				assert(support == VK_ERROR_FORMAT_NOT_SUPPORTED);
```

Comment:
`verify(HERE), support == VK_ERROR_FORMAT_NOT_SUPPORTED;`

Reply:
Add an error message `LOG_ERROR(RSX, "Format 0x%x is not supported for rendertarget usage by your GPU driver. Crashes may arise.", format);`